bundles/org.openhab.binding.wled/README.md

```diff
@@ -0,0 +1,96 @@
+# WLED Binding
+
+This openHAB binding allows you to auto discover and use LED strings based on the WLED project:
+<https://github.com/Aircoookie/WLED>
+
+## Fault Finding
+
+To watch what the binding does you can enter this in to the openHAB console, `log:set TRACE org.openhab.binding.wled` which will allow you to test the same commands in a web browser to determine if it is a bug in the binding, or in the firmware for WLED.
+Firmware 0.10.2 is working very well with this binding after extensive testing, so if an issue is found please report what firmware version you are using.
+
+## Supported Things
+
+| Thing Type ID | Description |
+|-|-|
+| `wled` | Use this for RGB and RGBW strings. |
+
+## Discovery
+
+The auto discovery will work with this binding if your network supports mDNS.
+If it fails to find your WLED, you can manually add a `wled` thing by using the UI or textual methods.
+The full example section below gives everything needed to quickly setup using textual config.
+
+## Thing Configuration
+
+| Parameter | Description |
+|-|-|
+| `address`| The full URL to your WLED device. Example is `http://192.168.0.2:80` |
```

Comment:
This project keeps adding a huge number of features, a kitchen sink is probably planned. Https has been mentioned as possible and planned in ESP32, so whilst it appears to not have the support yet, it is safer to assume at some point it will get added.

1. mDNS reports the URL that is in the example so no need to change it from what the auto discovery gives.
2. I use hostnames as much as possible instead of IP address'
3. I have already made the binding accept a plain IP Address should a user enter one in. Both formats work.

I will remove the tag that you suggested in another comment.

Reply:
It's a shame that you can't use the UI validation, then. But I don't have a better solution.